tests/coverage/run/i18233-min/i182233-min.check

```diff
@@ -0,0 +1,2 @@
+List()
```

Comment:
Is this name also wrong btw? i182233-min.check has an added 2

Reply:
Yes. Weird that the added assertion does not throw... I'll try to look at this closer tomorrow

Reply:
Ah I see the issue - for that one, the first file returned by the iterator is actually `i182233-min.check` so after replacing .scala with .check, we end up with the same file, so it passes the assertion (and is indeed used in the run test).

Reply:
Ach, so not a big deal then